cosmwasm/utils.js

```diff
@@ -114,6 +118,17 @@ const getAmplifierContractConfig = (config, { contractName, chainName }) => {
     return { contractBaseConfig, contractConfig };
 };
 
+const usesGovernanceBypass = (config, contractName, chainName) => {
```

Comment:
this implicit switch can be confusing for UX (what if the mnemonic is not actually the right one). Instead let the user control it by either providing `--governance` or not to simplify? By default, EOA is used unless user overrides.